.goreleaser.yml

```diff
@@ -1,9 +1,65 @@
 builds:
-- 
+-
   goos:
   - windows
   - darwin
   - linux
   goarch:
     - amd64
-    - arm64
\ No newline at end of file
+    - arm64
+
+# .goreleaser.yaml
+release:
+  # Repo in which the release will be created.
+  # Default is extracted from the origin remote URL or empty if its private hosted.
+  github:
+    owner: great-fork
+    name: json2hcl
+
+  # IDs of the archives to use.
+  # Defaults to all.
```

Comment:
Is this comment relevant since there is no property below it?

Reply:
I left it here in case we can use it in the future, but we can also only keep the settings we use.

How do you think?

Reply:
Good thinking, but I would remove it for now.